master/internal/db/postgres_checkpoints.go

```diff
@@ -0,0 +1,86 @@
+package db
+
+import (
+	"fmt"
+
+	"github.com/google/uuid"
+)
+
+// FilterForRegisteredCheckpoints gets the deleted checkpoints provided in the model registry.
+func (db *PgDB) FilterForRegisteredCheckpoints(deleteCheckpoints []uuid.UUID) ([]uuid.UUID, error) {
```

Comment:
`deleteCheckpoints` -> `checkpoints` or `cs`.. this method has no idea they're being deleted.

Reply:
also the comment implies the checkpoints passed are deleted.